src/neural/blas/convolution1.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ This file is part of Leela Chess Zero.
+ Copyright (C) 2018 The LCZero Authors
+
+ Leela Chess is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Leela Chess is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "neural/blas/convolution1.h"
```

Comment:
sorted includes. :)
But clang-format would do that for you.

Reply:
> There are many things like this already (like policy head size). Maybe have network-common.h ?
Actually no. There are not many things like this. We figure network parameters using weights vector sizes. Maybe the number of input planes is the exception? Even the number of policies is computed:

```
const auto num_output_policy = weights_.ip_pol_b.size();
``

Reply:
> sorted includes. :)
> But clang-format would do that for you.

I will pass clang-format on the final result.